packages/js/product-editor/src/components/label/label.tsx

```diff
@@ -0,0 +1,45 @@
+/**
+ * External dependencies
+ */
+import { createElement } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+import { Icon, help as helpIcon } from '@wordpress/icons';
+import { Tooltip } from '@wordpress/components';
+
+interface LabelProps {
+	label: string;
+	required?: boolean;
+	tooltip?: string;
+}
+
+export const Label: React.FC< LabelProps > = ( {
```

Comment:
I've seen us use this `React.FC< LabelProps >` typing before, but I don't see how this really buys us anything. I did some quick searching and found that some other individuals also think that `React.FC` shouldn't be used.

https://kentcdodds.com/blog/how-to-write-a-react-component-in-typescript

I don't feel super strongly about this, since we do this all over the place currently.

Reply:
Interesting. I will leave it like this for now since it's not affecting anything, and it also handles the return type, but I agree it shouldn't be enforced.